changelog/unreleased/enhancement-add-events-setting.md

```diff
@@ -0,0 +1,6 @@
+Enhancement: The user settings was extended by the notifications preferences
+
+The user settings was extended, that a user can store their notifications preferences
+
+https://github.com/owncloud/ocis/pull/10706  
+https://github.com/owncloud/ocis/issues/10669
```

Comment:
```suggestion
Enhancement: Part I: The user settings are extended by notifications preferences

Part I: Users can now store their notifications preferences in the user settings

https://github.com/owncloud/ocis/pull/10706  
https://github.com/owncloud/ocis/issues/10669
```
This is the same changelog text as in #10779 - which I just reviewed (created two suggestable commits)
I do not think that this is a good idea to have two identical changelog entries.
This small change makes this distinguishable but anyone can see it belongs together (part I / part II)